package.json

```
@@ -25,6 +25,7 @@
     "testonly-start-turbo": "cross-env NEXT_TEST_MODE=start TURBOPACK=1 TURBOPACK_BUILD=1 pnpm testonly",
     "testonly-deploy": "cross-env NEXT_TEST_MODE=deploy pnpm testonly",
     "test": "pnpm testheadless",
+    "test-turbo": "TURBOPACK=1 TURBOPACK_DEV=1 TURBOPACK_BUILD=1 pnpm testheadless",
```

```suggestion
    "test-turbo": "cross-env TURBOPACK=1 TURBOPACK_DEV=1 TURBOPACK_BUILD=1 pnpm testheadless",
```

Should use `cross-env`